lib/variant-model.js

```diff
@@ -59,6 +59,20 @@ _.extend(Variant.prototype, {
     });
   },
 
+  respondWithVariantFile: function(mockDir, fileName) {
+    var self = this;
+    return self.getJsonResponse(self._id, self._route._path, mockDir, fileName);
+  },
+
+  getJsonResponse: function(variant, path, mockDir, fileName) {
```

Comment:
why are you passing variant id?  it's an attribute of the variant model (which this is a prototype method of).  am I missing something?